config/service-brokers/compose/catalog.json

```diff
@@ -27,5 +27,33 @@
         "units": 1
       }
     }]
+  },{
+    "id": "6e9202f2-c2e1-4de8-8d4a-a8c898fc2d8c",
+    "name": "elastic_search",
```

Comment:
This should not have an underscore in it. Elasticsearch is not two words - https://www.elastic.co/products/elasticsearch

Spelling it like this will only add friction for users.